src/robot/output/pyloggingconf.py

```diff
@@ -65,11 +68,11 @@ def emit(self, record):
         method = self._get_logger_method(record.levelno)
         method(message)
         if error:
-            librarylogger.debug(error)
+            self.library_logger.debug(error)
 
     def _get_message(self, record):
         try:
-            return record.getMessage(), None
+            return self.format(record), None
```

Comment:
This is the actual fix, right?

Reply:
Yup, that's the fix that causes messages to now be passed through the set formatter